dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/MetricAdapter.java

```diff
@@ -341,10 +345,15 @@ static String toMintMetricKey(String metricKey) throws DynatraceExporterExceptio
    * @param key is the key as String.
    * @param value is the value as String.
    * @return a MINT-compatible Dimension (sanitized key and value).
-   * @throws DynatraceExporterException if an error occured during sanitizing the Strings.
+   * @throws DynatraceExporterException if an error occurred during sanitizing the Strings or if a
+   *     null value was passed.
    */
   static Dimension toMintDimension(String key, String value) throws DynatraceExporterException {
-    return Dimension.create(toMintDimensionKey(key), toMintDimensionValue(value));
+    try {
+      return Dimension.create(toMintDimensionKey(key), toMintDimensionValue(value));
+    } catch (NullPointerException npe) {
```

Comment:
Explicitly null-checking both key and value would be more efficient than having an additional exception thrown.